tools/setup_env.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/bash -e
-gcloud components install app-engine-java
-
-cd ui
-echo Installing the Angular command-line tool, to serve/build the UI.
-sudo npm install -g @angular/cli
-echo Installing UI project dependencies.
-npm install
-
-
-cd ../tools
-./download_swagger_cli.sh
-./download_cloud_sql_proxy.sh
```

Comment:
I would like to see some documentation for where we are going to use this script. I think for now it may make sense to keep `setup_env.sh`, and just remove pieces of it as they become obsolete. 

I am comfortable getting rid of the npm install pieces, and putting those as setup instructions in the readme. (I believe they already may be)